### DISCUSSION TITLE
Clapping is being transcribed as thank you

### DISCUSSION THREAD

**Discussion by kanishknishar** [openai/whisper#905] [Q&A]
I have been transcribing a lot of plays recently which end with the audience clapping and it is consistently transcribed as "thank you" for some reason. 

Build: 17-01-2023
Model: small.en

I can share my video files if needed.

**Comment by glangford** [comment]
Maybe the training data had a lot of examples where an audience was clapping, and the presenter/speaker said "Thank you" in response to the applause? Might be worth trying on medium or large model.

**Comment by EtienneAb3d** [comment]
Certainly because Whisper was trained with a lot of video sounds / subtitles where a "Thank you" text was added in the subtitle at the end, when clapping was in the sound.
Read #679
